wiki/Contests/en.md

```diff
@@ -2,291 +2,420 @@
+| [#4 (2017)](https://osu.ppy.sh/home/news/2017-08-14-osu-idol-2017-auditions-now-open) | ![][flag_AU] [Renril](https://osu.ppy.sh/users/4955122) | ![][flag_US] [Will Stetson](https://osu.ppy.sh/users/4909088) | ![][flag_US] [Thievley](https://osu.ppy.sh/users/4717672) | [Renril - Rinne Tensei](https://assets.ppy.sh/contests/57/entries/Renril_-_Rinne_Tensei.mp3) | [Results](https://osu.ppy.sh/home/news/2017-12-03-osu-idol-2017-finals-community-voting-results) |
+| [#5 (2018)](https://osu.ppy.sh/home/news/2018-07-01-osu-idol-2018-auditions-now-open) | ![][flag_US] [Will Stetson](https://osu.ppy.sh/users/4909088) | ![][flag_ID] [Zeva\[FL\]](https://osu.ppy.sh/users/2805170) | ![][flag_US] [MjIsMaster](https://osu.ppy.sh/users/9922896) | [Will Stetson - Snow Drive](https://assets.ppy.sh/contests/71/entries/Snow%20Drive.mp3) | [Results](https://osu.ppy.sh/home/news/2018-11-05-osu-idol-2018-finals-community-voting-results) |
+| [#6 (2019)](https://osu.ppy.sh/home/news/2019-07-02-osu-idol-2019-auditions-now-open) | ![][flag_PH] [iamALieee](https://osu.ppy.sh/users/10093980) | ![][flag_US] [aceredshirt13](https://osu.ppy.sh/users/13736238) | ![][flag_US] [moof](https://osu.ppy.sh/users/8457202) | [iamALieee - White First Love](https://assets.ppy.sh/contests/83/entries/osu!idol%202019%20-%20iamALieee%20-%20White%20First%20Love.mp3) | [Results](https://osu.ppy.sh/home/news/2019-12-21-osu-idol-2019-finals-community-voting-results) |
+| [#7 (2020)](https://osu.ppy.sh/home/news/2020-09-01-osu-idol-2020-auditions-now-open) | *N/A* | *N/A* | *N/A* | *N/A* | *N/A* |
+
+#### osu!remix
+
+osu!remix is a song remix contest where participating musicians from all across the community are being tasked to the best remix of a song that has been widely known throughout the osu! community in one way or another. It is the first ever contest to be handled via the [Community Contests listing](https://osu.ppy.sh/community/contests) page.
+
+| Contest No. | Remixed Song | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
+| :-: | :-- | :-- | :-- | :-- | :-- |
+| [#1](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest) | [nekodex - circles!](https://osu.ppy.sh/beatmaps/artists/1) | ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) | ![][flag_FR] [Kurokotei](https://osu.ppy.sh/users/398275) | ![][flag_NL] [happy30](https://osu.ppy.sh/users/27767) | [Results](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) |
+
+#### Skinning Contest
+
+The Skinning Contest is a skinning contest open to skin creators from all across the community. 
```

Comment:
this is very vague? may be worth putting a bit more detail in here

Reply:
```suggestion
The **Skinning Contest** is a skinning contest where artistic creators from all across the community compete to create the best skin in a limited time. 
```

Reply:
going with this yup